Tasks/AzureFileCopyV5/README.md

```diff
@@ -0,0 +1,118 @@
+# Azure File Copy
+
+### Overview
+
+The task is used to copy application files and other artifacts that are required to install the application on Azure VMs like PowerShell scripts, PowerShell-DSC modules etc. The task provides the ability to copy files to an Azure blob or directly to Azure VMs. Even when the target is Azure VMs, Azure blobs are used as an intermediary and the files are copied to it first and then downloaded to the VMs. The tasks uses [AzCopy](https://azure.microsoft.com/en-in/documentation/articles/storage-use-azcopy/), the command-line utility built for fast copying of data from and into Azure storage accounts.
+
+### Contact Information
+
+Please report a problem at [Developer Community Forum](https://developercommunity.visualstudio.com/spaces/21/index.html) if you are facing problems in making this task work.  You can also share feedback about the task like, what more functionality should be added to the task, what other tasks you would like to have, at the same place.
+
+### Prerequisite for the task
+
+**Azure Subscription**
+
+To deploy to Azure, an Azure subscription has to be linked to Team Foundation Server or to Azure Pipelines using the Services tab in the Account Administration section. Add the Azure subscription to use in the Build or Release Management definition by opening the Account Administration screen (gear icon on the top-right of the screen) and then click on the Services Tab. 
+
+ - Use 'Azure Resource Manager' endpoint type to create a ARM endpoint, for more details follow the steps listed in the link [here](https://go.microsoft.com/fwlink/?LinkID=623000&clcid=0x409).
+ -  As this version of task uses AzCopy 10, service principal needs to have one of these `Storage Blob Data Contributor` or `Storage Blob Data Owner` roles assigned to access resources. 
+    1. Click on `Manage` link next to Azure Subscription
+    2. Click `Manage Service Principal` which will redirect you to the Application Registration of the Service Principal. Copy the name.
+    3. Go back and click `Manage service connection roles` which will redirect you to the IAM blade of the Azure Subscription. Here you need to assign a role to the service principal. Use name copied in previous step to search service principal  
+
+**PowerShell**
+
+The task needs at least version 3.0 of Powershell on the target machine.
+
+**Azure PowerShell**
+
+The task needs the Azure PowerShell version to be installed on the automation agent, and that can be done easily using the [Azure PowerShell Installer v1.0.2] (https://github.com/Azure/azure-powershell/releases/tag/v1.0.2-December2015). Refer to "Supported Azure and Az module versions" section below for recommended versions.
```

Comment:
Change this to "this task needs Az modules to be installed on the automation agent, and that can be done easily using https://docs.microsoft.com/en-us/powershell/azure/install-az-ps?view=azps-7.0.0"

Reply:
done.